bagua/script/baguarun.py

```diff
@@ -67,6 +67,8 @@ def pssh_bagua_launch(
         )  # noqa: E501
     if args.no_python:
         bypass_args.append("--no_python")
+    if args.enable_bagua_net:
+        bypass_args.append("--enable-bagua-net")
```

Comment:
Why not make it `--enable_bagua_net` to be consistent with other options' style?